ci/test_integration.sh

```diff
@@ -17,15 +17,4 @@
 #!/bin/bash
 set -e
 
-# Get latest models version
-cd /models/
-git pull origin main
-
-container=$1
-
-## Tensorflow training container
-if [ "$container" == "merlin-tensorflow-training" ]; then
-    ## TODO: Remove this when it would become part of the container
-    pip3 install testbook
```

Comment:
do we have testbook and jupyterlab  on the container now ?

Reply:
no they don't currently.